specs/v2-sketch.yaml

```diff
@@ -100,6 +100,42 @@ paths:
               operationId: getProjectById
               parameters:
                 project_slug: "$response.body#/project_slug"
+  /pipeline/{id}/config:
+    get:
+      summary: Return a pipeline's configuration
```

Comment:
I've chosen to have a single route that returns pre- and post-compile config. I don't feel strongly about this vs a separate route for each.

Reply:
+1 for one-for-both.